Add elastic-agent diagnostics documentation (#1164)
elastic/observability-docs: docs/en/ingest-management/commands.asciidoc

* <<elastic-agent-uninstall-command,uninstall>>
* <<elastic-agent-upgrade-command,upgrade>>
* <<elastic-agent-version-command,version>>

> * <<elastic-agent-diagnostics-command,diagnostics>>

Reviewer: These commands are alphabetized, so move this one to the top.

<hr>
++++

> [discrete]

Reviewer: Same comment as before. Move this to the top section because it appears first alphabetically.

[[elastic-agent-diagnostics-command]]
== elastic-agent diagnostics

Gather diagnostics information from the {agent} and applications it's running.

> Will display version numbers and application meta-data.

Reviewer: ```suggestion
This command displays version numbers and application metadata.
```

[discrete]
=== Synopsis

[source,shell]
----
elastic-agent diagnostics [--help] [--output <string>] [global-flags]
----

[discrete]
=== Options

`--output <string>`::

> Output format, default "human". Must be one of human, json, yaml.

Reviewer: ```suggestion
Output format. Specify `human`, `json`, or `yaml`. The default is `human`.
```
